bin/hermit.hcl

```diff
@@ -1,6 +1,6 @@
 sources = ["https://github.com/cashapp/hermit-packages.git", "https://github.com/foxygoat/hermit-packages.git"]
 env = {
   GOBIN: "${HERMIT_ENV}/out/bin",
-  PATH: "${GOBIN}:${PATH}",
+  PATH: "${GOBIN}:${HERMIT_ENV}/scripts:${PATH}",
```

Comment:
you were asking somewhere for feedback on this. i think it is good.
i run for example `servedir` regularly - as quite often i don't want to rebuild all deps (to lazy to wait hasn't stung me much yet i think). 
so i think it's fine.

Reply:
I dont understand the `servedir` comment. That is go-installed, and wouldn't be in the `scripts/` directory.

Reply:
It is to say: personally I don't drive everything from the makefile

Reply:
as discussed, i've dropped this from the path (dropped the first commit). If it turns out it might be useful based on use/experience we can add it back.